subsys/bluetooth/controller/Kconfig

```diff
@@ -200,6 +207,13 @@ config BLUETOOTH_CONTROLLER_SCHED_ADVANCED
 	  Disabling this feature will lead to overlapping role in timespace
 	  leading to skipped events amongst active roles.
 
+config BLUETOOTH_CONTROLLER_TIFS_HW
```

Comment:
One, due to the nature of NRF_RADIO->PACKETPTR being common to Rx and Tx, using the hw switching will still need timers+ppis to watch(dog) whether ISR has run on time, to avoid packet buffer corruption and/or spurious transmissions.
Two, Kconfig option enables application developer in making the choice, lower power hard realtime vs slightly higher power consumption due to use of (watchdog) timer to skip events under occasional latencies on Radio ISR.